src/components/AssignmentSummary.jsx

```diff
@@ -106,6 +107,77 @@ export class AssignmentSummary extends Component {
     }
   }
 
+  calcHexBrightness(color) {
+    let hexcolor = color.replace("#", "");
+    if (hexcolor.length === 3) {
+      hexcolor = hexcolor.replace(/(.)/g, "$1$1");
+    }
+    const r = parseInt(hexcolor.substr(0, 2), 16);
+    const g = parseInt(hexcolor.substr(2, 2), 16);
+    const b = parseInt(hexcolor.substr(4, 2), 16);
+
+    const brightness = Math.floor((r * 299 + g * 587 + b * 114) / 1000);
```

Comment:
What are the numbers `299`, `587` and `114` used for? Could those be variables instead to make it clearer why we're doing that multiplication?